src/opera_disp_tms/create_tile_map.py

```diff
@@ -1,14 +1,34 @@
 import argparse
+import json
 import multiprocessing
 import subprocess
 import tempfile
 
-from osgeo import gdal, gdalconst
-
+from osgeo import gdal, gdalconst, osr
 
 gdal.UseExceptions()
 
 
+def get_tile_bounds(info: dict, output_folder: str) -> None:
+    """Generate file with the bounds of the newly created vrt
+        Will return a file with: {"BOUNDS": [minx, miny, maxx, maxy], "EPSG": %EPSG}
```

Comment:
```suggestion
def get_tile_bounds(info: dict, output_folder: Path) -> None:
    """Generate file with the bounds of the newly created vrt
        Will return a file with: {"extent": [minx, miny, maxx, maxy], "EPSG": %EPSG}

```

Reply:
We should use a `Path` object here, it's just a bit more resilient than using strings.